recipes/fenicsx-beat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "fenicsx-beat" %}
+{% set name_underscore = "clang_format_docs" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name_underscore }}-{{ version }}.tar.gz"
+  sha256: 171d26a2e1ac6dfed4ff7c267db27e239c16bac306dfc8b1d142f714d56270d6
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
I guess the linter would prefer you leave this out and ignore irrelevant Windows failures. That seems weird, but 🤷 .

```suggestion
```